pkg/kubernetes/pod/v1alpha1/build.go

```diff
@@ -40,6 +40,16 @@ func NewBuilder() *Builder {
 	return &Builder{pod: &Pod{object: &corev1.Pod{}}}
 }
 
+// WithTolerations sets the Spec.Tolerations withh provided value.
```

Comment:
Add a new function called - `WithTolerationsForTaints( taints ...) ..`, which will accept the taints and add them as tolerations.